### ISSUE TITLE
Crash in 1.19.60

### ISSUE THREAD

**Issue by Atara6431** [fuyutsuki/Texter#136]
```bash
PocketMine-MP Crash Dump Tue Feb 14 20:56:33 AWST 2023

PocketMine-MP version: 4.14.0 [Protocol 567]
Git commit: c7930ce9ec0045e712d21a65b95a24be6e051c8c
PHP version: 8.0.13
OS: WINNT, win

A PLUGIN WAS INVOLVED IN THIS CRASH
BAD PLUGIN: Texter

Error: pocketmine\network\mcpe\protocol\UpdateAbilitiesPacket::create(): Argument #1 ($data) must be of type pocketmine\network\mcpe\protocol\types\AbilitiesData, int given, called in phar://D:/BaiduNetdiskDownload/three/plugins/Texter (1).phar/src/jp/mcbe/fuyutsuki/Texter/text/FloatingText.php on line 146
File: pmsrc/vendor/pocketmine/bedrock-protocol/src/UpdateAbilitiesPacket
Line: 32
Type: TypeError
Backtrace:
#0 plugins/Texter (1).phar/src/jp/mcbe/fuyutsuki/Texter/text/FloatingText(146): pocketmine\network\mcpe\protocol\UpdateAbilitiesPacket::create(int 0, int 0, int 610, array[0])
#1 plugins/Texter (1).phar/src/jp/mcbe/fuyutsuki/Texter/text/FloatingText(194): jp\mcbe\fuyutsuki\Texter\text\FloatingText->asPackets(object pocketmine\player\Player#47170, object jp\mcbe\fuyutsuki\Texter\text\SendType#153869)
#2 plugins/Texter (1).phar/src/jp/mcbe/fuyutsuki/Texter/text/FloatingTextCluster(96): jp\mcbe\fuyutsuki\Texter\text\FloatingText->sendToPlayer(object pocketmine\player\Player#47170, object jp\mcbe\fuyutsuki\Texter\text\SendType#153869)
#3 plugins/Texter (1).phar/src/jp/mcbe/fuyutsuki/Texter/task/SendTextsTask(37): jp\mcbe\fuyutsuki\Texter\text\FloatingTextCluster->sendToPlayer(object pocketmine\player\Player#47170, object jp\mcbe\fuyutsuki\Texter\text\SendType#153869)
#4 pmsrc/src/scheduler/TaskHandler(123): jp\mcbe\fuyutsuki\Texter\task\SendTextsTask->onRun()
#5 pmsrc/src/scheduler/TaskScheduler(141): pocketmine\scheduler\TaskHandler->run()
#6 pmsrc/src/plugin/PluginManager(530): pocketmine\scheduler\TaskScheduler->mainThreadHeartbeat(int 834755)
#7 pmsrc/src/Server(1803): pocketmine\plugin\PluginManager->tickSchedulers(int 834755)
#8 pmsrc/src/Server(1692): pocketmine\Server->tick()
#9 pmsrc/src/Server(1053): pocketmine\Server->tickProcessor()
#10 pmsrc/src/PocketMine(339): pocketmine\Server->__construct(object BaseClassLoader#5, object pocketmine\utils\MainLogger#2, string[30] D:\BaiduNetdiskDownload\three\, string[38] D:\BaiduNetdiskDownload\three\plugins\)
#11 pmsrc/src/PocketMine(362): pocketmine\server()
#12 pmsrc(11): require(string[74] phar://D:/BaiduNetdiskDownload/three/PocketMine-MP.phar/src/PocketMine.php)

Code:
[23]  */
[24] class UpdateAbilitiesPacket extends DataPacket implements ClientboundPacket{
[25] 	public const NETWORK_ID = ProtocolInfo::UPDATE_ABILITIES_PACKET;
[26] 
[27] 	private AbilitiesData $data;
[28] 
[29] 	/**
[30] 	 * @generate-create-func
[31] 	 */
[32] 	public static function create(AbilitiesData $data) : self{
[33] 		$result = new self;
[34] 		$result->data = $data;
[35] 		return $result;
[36] 	}
[37] 
[38] 	public function getData() : AbilitiesData{ return $this->data; }
[39] 
[40] 	protected function decodePayload(PacketSerializer $in) : void{
[41] 		$this->data = AbilitiesData::decode($in);
[42] 	}
```

**Comment by xxFLORII** [comment]
Should be fixed here: https://github.com/fuyutsuki/Texter/tree/dev/1.19.60

**Comment by Atara6431** [comment]
please add download file in github, poggit.pmmp.io can't enter.

**Comment by fuyutsuki** [comment]
I know that, please wait until 4.1.9.

**Comment by fuyutsuki** [comment]
Hi, just released 4.1.9, please try it.
